src/module/custom.rs

```diff
@@ -29,6 +30,16 @@ pub trait CustomSection: WalrusAny + Debug + Send + Sync {
     /// payload. `walrus` will handle these for you.
     fn data(&self, ids_to_indices: &IdsToIndices) -> Cow<[u8]>;
 
+    /// Add any core wasm roots to the provided `roots` argument.
+    ///
+    /// This function will add any referenced core wasm items into the `Roots`
+    /// array provided.
+    ///
+    /// The default provided method does nothing.
+    fn gc_add_roots(&self, roots: &mut Roots) {
```

Comment:
nitpick: let's name this `add_gc_roots`, since that makes more sense when read out loud in a sentence.